### DISCUSSION TITLE
How to skip larger testcases (~800kb) that cause a timeout?

### DISCUSSION THREAD

**Discussion by warlock2k** [AFLplusplus/AFLplusplus#1752] [General]
Hello,

Some of the test cases in my corpus are too large, how do I ensure that these are skipped during the next fuzzrun, so that my fuzzer doesn't timeout? If I do not skip, it makes the overall execution too slow as I set timeout to ~5000ms

**Comment by vanhauser-thc** [comment]
simply delete them.
`find corpus/ -size +500k -exec rm {} \;`

**Comment by warlock2k** [comment]
@vanhauser-thc Thank you. Is there a way to skip these testcases instead during the run? Does providing -t 1000+ work?

**Reply by vanhauser-thc** [reply]
you cannot skip these during the run. but if they are too slow just set a lower timeout and then they will not be used.
what you can also do is creating a simple custom mutator (see docs/custom_mutators.md and custom_mutators/examples/*) and use `afl_custom_queue_get`. return 0 to not fuzz a queue entry (or even better, setting it's queue entry to `disabled = 1` but that is more advanced).

**Reply by warlock2k** [reply]
Perfect, I will use this.

**Comment by warlock2k** [comment]
Also, could you kindly tell me why is that afl slows down upon increasing the timeout? Is it because the mutated outputs also remain larger?

**Reply by vanhauser-thc** [reply]
well the higher the timeout you set, the the longer it takes for a hang or timeout to be hit, so naturally this slows down fuzzing. kinda obvious right?

**Reply by warlock2k** [reply]
Makes sense, thanks!

**Comment by warlock2k** [comment]
@vanhauser-thc I am also assuming that if I use the plus sign, then afl won't be classifying hangs?

**Reply by vanhauser-thc** [reply]
no that is wrong. read the docs what the + sign means
